test_utils/__init__.py

```diff
@@ -13,7 +13,7 @@
 TEST_LMS_USER_ID = 1
 
 
-class ProgramIntentsAPITestCase(JwtMixin, APITestCase):
+class ProgramIntentAPITestCase(JwtMixin, APITestCase):
```

Comment:
Nit: Is this from the rebase onto Varsha's code? I'm just checking if you actually need to rename this.